ArcGISRuntimeSDKQt_CppSamples/Scenes/FeatureLayerExtrusion/FeatureLayerExtrusion.cpp

```diff
@@ -0,0 +1,128 @@
+// [WriteFile Name=FeatureLayerExtrusion, Category=Scenes]
+// [Legal]
+// Copyright 2017 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "FeatureLayerExtrusion.h"
+
+#include "FeatureLayer.h"
+#include "ServiceFeatureTable.h"
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "OrbitLocationCameraController.h"
+#include "Point.h"
+#include "SimpleLineSymbol.h"
+#include "SimpleFillSymbol.h"
+#include "SimpleRenderer.h"
+#include "RendererSceneProperties.h"
+
+
+using namespace Esri::ArcGISRuntime;
+
+bool showTotalPopulation = true;
+double distance = 20000000;
+double heading = 0;
+double pitch = 55;
+double roll = 0;
+
+FeatureLayerExtrusion::FeatureLayerExtrusion(QQuickItem* parent /* = nullptr */):
+    QQuickItem(parent)
+{
+}
+
+void FeatureLayerExtrusion::init()
+{
+    // Register classes for QML
```

Comment:
throughout our cpp/h files, we use 2 space indent. I probably missed this on the shapefile sample. Easiest way to change this is to go into Qt Creator Settings > C++ > change to 2 space indent and save that style. Then select all the code in the cpp/h file and hit ctrl + i, and it will auto format

Reply:
Great tip. Thanks